crowdsec-docs/docs/appsec/quickstart.md

```diff
@@ -4,45 +4,62 @@ title: Quickstart
 sidebar_position: 2
 ---
 
-<!-- @jdv deprecated, or we link it in installation tutorial (renaming it too) but lets not have dup content -->
+## Objectives
 
-The AppSec installation and configuration are pretty straightforward.
+The goal of this quickstart is to set up the [AppSec Component](/appsec/intro.md#introduction) to safeguard web applications running on [Nginx](https://nginx.com). We'll deploy a [set of rules](https://app.crowdsec.net/hub/author/crowdsecurity/collections/appsec-virtual-patching) designed to block well-known and [currently exploited vulnerabilities](https://app.crowdsec.net/hub/author/crowdsecurity/collections/appsec-virtual-patching). Additionally, we'll show how to monitor these alerts through the [console](https://app.crowdsec.net/).
 
-You can find a more advanced installation documentation [here](/appsec/installation.md).
+## Pre-requisites
 
-To make the AppSec component work, we need to follow 3 steps.
+1. If you're new to the [AppSec Component](/appsec/intro.md#introduction) or **W**eb **A**pplication **F**irewalls, start with the [Introduction](/appsec/intro.md#introduction) for a better understanding.
 
-### Install the collection
+2. It's assumed that you have already installed:
+   - **Crowdsec [Security Engine](/docs/next/intro)**: for installation, refer to the [QuickStart guide](/docs/getting_started/install_crowdsec). The AppSec Component, which analyzes HTTP requests, is included within the security engine.
+   - Nginx and its **[Remediation Component](/u/bouncers/intro)**: installation instructions are available in the [QuickStart guide](/u/bouncers/nginx). This component intercepts HTTP requests at the webserver or reverse-proxy level and forwards them to the AppSec Component for analysis and action.
```

Comment:
Do we want to say that's available for openresty too ? And what do we plan when we'll support other server  web ?

Reply:
I think we'll wait to have a bit more and publish a compatibility matrix :)

Reply:
Currently:

Done:
- Traefik
- Nginx
- OpenResty

WIP:
- Caddy

No started:
- Apache
- HAProxy